modules/react/modal/stories/stories_VisualTesting.tsx

```diff
@@ -57,11 +61,13 @@ const TestModal = () => {
 export const ModalSmallWidth = withSnapshotsEnabled(() => <TestModal />);
 
 export const ModalRTL = withSnapshotsEnabled(() => {
+  const {direction, ...rest} = defaultCanvasTheme;
```

Comment:
Curious of anyone has thoughts on this approach. The styling of the modal depends on `theme`. If someone only provides a partial theme to override direction, the rest of the canvas theme isn't provide and this causes the component to not render because it doesn't know what breakpoints is. I had to import defaultCanvasTheme and just change direction and then still spread the rest of the theme into the provider. CC @NicholasBoll @alanbsmith

Reply:
Have you tried: `const theme = useTheme({canvas: {direction: ContentDirection.RTL}});` and pass it into `CanvasProvider`? I guess it should create a the same theme but only with changed direction.

Reply:
YUS! thank you!!